src/index.html

```diff
@@ -6,9 +6,97 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+
+      <a class ="logo" href="#">
+        <img
+          class="logo__img"
+          src="images/logo.png"
+          alt="logo"
+        >
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a class="nav__link is-active" href="#">
+            APPLE
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#"
+            >
+            SAMSUNG
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#"
+            >
+            SMARTPHONES
+            </a>
+          </li>
+
+          <li class="nav__item"
+            data-qa="hover">
+            <a
+              class="nav__link"
+              href="#"
+            >
+              <span class="link_four">
+              LAPTOPS & COMPUTERS
+              </span>
+            </a>
+          </li>
```

Comment:
1. hover should be applied to a link
2. code styling (check recommendations from a [checklist](https://github.com/mate-academy/layout_moyo-header/blob/master/checklist.md)
3. span tag is redundant
```suggestion
          <li class="nav__item">
            <a
              data-qa="hover"
              class="nav__link"
              href="#"
            >
              laptops & computers
            </a>
          </li>
```